api/RTCPeerConnection.json

```diff
@@ -2523,6 +2526,2019 @@
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": "56"
+            },
+            "chrome_android": {
+              "version_added": "56"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "43"
+            },
+            "opera_android": {
+              "version_added": "43"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "ontrack": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/ontrack",
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": "56"
+            },
+            "chrome_android": {
+              "version_added": "56"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "43"
+            },
+            "opera_android": {
+              "version_added": "43"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "addIceCandidate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/addIceCandidate",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "50",
+                "notes": "Promise based version.",
+                "version_removed": "56"
+              },
+              {
+                "version_added": "56"
+              }
```

Comment:
I was up until 2am fixing this file. The final commit 34cae58587abd1c65bc40e124b47f75b2b42a925 is only 2537 lines long. The review you did was unfortunately on one of the 2 previously failing commits in Travis.

Reply:
When looking at your review vs what is in the file, they match up. 
![screenshot from 2018-02-15 08-06-11](https://user-images.githubusercontent.com/5793295/36246354-5889a35c-1227-11e8-916e-2b48a8a30d66.png)

Reply:
The [link you shared](https://www.chromestatus.com/features/5743313768415232) discussing M50.. 
>Add promise-based versions of RTCPeerConnection methods: setLocalDescription, setRemoteDescription, addIceCandidate, createOffer and createAnswer. To be done in 2 steps. First, setLocalDescription, setRemoteDescription and addIceCandidate (anticipated in M50).

Reply:
Under the light of a new day this now does not look quite right, as it suggests the removal of promises in 56. Seeking out another potential code example inside BCD to replicate, to avoid unecessary repetitious notes in a fix.

Reply:
Adding the line `"notes": "Promise based version and unprefixed."` to an immediate commit. This will communicate promises did not STOP in 56, in all instances of earlier rolled out subfeatures.